joomla/joomla-cms
# [4.0] Drop MSSQL Install and Update SQL Files

### Summary of Changes

This PR just removes the update and install SQL files for MSSQL based on: https://developer.joomla.org/news/658-joomla4-manifesto.html

### Testing Instructions

- Code review
- make sure that install on mssql don't work (as there are no install SQL files anymore)

### Documentation Changes Required

Drop of MS SQL support
